UIKitDZ/UIKitDZ/ViewControllers/TimerViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  TimerViewController.swift
+//  UIKitDZ
+//
+//  Created by Анастасия Козлова on 02.10.2022.
+//
+
+import UIKit
+/// Экран таймер 
+final class TimerViewController: UIViewController {
+
+    @IBOutlet weak var cancellButton: UIButton!
```

Comment:
```suggestion
    @IBOutlet private weak var cancellButton: UIButton!
```
и везде проставить ниже где позволяет компилятор